### PR TITLE
[Enterprise-3.4]Updated OCP 3.4 Release Notes for RHSA-2017-3389

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

File: release_notes/ocp_3_4_release_notes.adoc [hunk 1]
```diff
@@ -1702,3 +1702,194 @@ openshift3/registry-console:3.4-30
 To upgrade an existing {product-title} 3.3 or 3.4 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-6-rhba-2017-3049-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 3.5 or 3.6 cluster to this latest
+release, use the automated upgrade playbook. See
+xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
```
Comment (adellape) [inline release_notes/ocp_3_4_release_notes.adoc]: I think this was a copy/paste from 3.6 content, needs version tweaks.

File: release_notes/ocp_3_4_release_notes.adoc [hunk 2]
```diff
@@ -1702,3 +1702,194 @@ openshift3/registry-console:3.4-30
...
+===== Security
+
+* An attacker with knowledge of the given name used to authenticate and access
+Elasticsearch can later access it without the token, bypassing authentication.
+This attack also requires that the Elasticsearch be configured with an external
+route, and the data accessed is limited to the indices.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1501986[*BZ#1501986*])
+
+[[ocp-3-4-rhsa-2017-3389-images]]
+==== Images
```
Comment (adellape) [inline release_notes/ocp_3_4_release_notes.adoc]: Not critical, but usually we have the image list right after the intro part (before the list of BZs) in this scenario, right?

**Review by adellape — CHANGES_REQUESTED** [review]

File: release_notes/ocp_3_4_release_notes.adoc [hunk 3]
```diff
@@ -1702,3 +1702,194 @@ openshift3/registry-console:3.4-30
...
+
+* A slow image pull made the network diagnostics fail. With this bug fix, the
+ timeout for the image pull was increased. The diagnostics now run in slow
+ environments.
+ (link:https://bugzilla.redhat.com/show_bug.cgi?id=1481550[*BZ#1481550*])
+
+* The OpenShift node proxy previously did not support using a specified IP
+address. This could prevent correct operation on hosts with multiple network
+interface cards. The OpenShift node process already accepts a
+`--bind-address=<ip address>:<port>` command-line flag and `bindAddress:`` configuration
```
Comment (adellape) [inline release_notes/ocp_3_4_release_notes.adoc]: Possible extra `` on `bindAddress` here.

**Comment by adellape** [comment]
@ahardin-rh Also needs the `revhistory_full.adoc` file updated.

File: release_notes/ocp_3_4_release_notes.adoc (hunk 1 above)
Comment (adellape) [reply release_notes/ocp_3_4_release_notes.adoc]: Oh actually this looks like it shouldn't be here at all maybe? From the previous section?

**Comment by ahardin-rh** [comment]
@adellape Thanks! Updated. I'll apply any similar changes to the other branch PRs

**Comment by ahardin-rh** [comment]
@adellape Also bumped latest-tags for each branch

**Review by adellape — APPROVED** [review]

**Comment by adellape** [comment]
:shipit:

**Pull request merged**